README.md

```diff
@@ -1,6 +1 @@
-# identity-play-auth
-A small library for Guardian Identity authentication with the Play framework
-
-* Use 2.x series for Play 2.6
-* Use 1.x series for Play 2.5
-* Use 0.x series for Play 2.4
+__This library is deprecated and should not be added to any projects. Identity are in the process of removing it from all projects currently using it.__ [identity-auth-play](https://github.com/guardian/identity) should be considered instead.
```

Comment:
I'm going to add a trello card for adding an `identity-auth-play` README

Reply:
Nice! any identity-auth-core?